var/spack/repos/builtin/packages/mel/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Mel(MakefilePackage):
+    """MPI implementation of Half-approximate Graph Matching"""
+
+    homepage = "https://github.com/ECP-ExaGraph/mel"
+    git = "https://github.com/ECP-ExaGraph/mel.git"
+
+    maintainers = ["Cgbrl28"]
+
+    version("develop", branch="master")
```

Comment:
From a user perspective (and a general rule), the version name should match the branch.